wiki/BanchoBot/en.md

```diff
@@ -1,154 +1,148 @@
+BanchoBot can respond to player commands through specific typed messages in chat. All BanchoBot commands start with an exlcamation mark (`!`) followed by the name of the command (with no space between). These commands can be used in multiplayer chat channels and through private messages with BanchoBot. 
 
-You can also use the `/bb` in-game client command to automatically open a tab with BanchoBot and send the command immediately.
+If a normal user sends a command in a public chat channel, other users will not see it, but the response will be displayed in a private message with BanchoBot. Users may also use the `/bb` in-game client command to automatically open a tab with BanchoBot and send the command immediately. A list of all BanchoBot commands can be found below:
 
-*Notice: The `REQUEST` command is not included because it is no longer supported.*
+*Note: The `!request` command is not included because it is no longer supported.*
 
-*Note: The user argument is case-insensitive, as usernames are case-insensitive.*
+- [Help](#help)
+- [Roll](#roll)
+- [Stats](#user-statistics)
+- [Where](#where)
+- [FAQ](#faq)
+- [Report](#report)
 
-### HELP
+### Help
 
 ```
-!HELP
+!help
 ```
 
-Shows a list of available BanchoBot commands.
-
-#### Usage Example
+`!help` shows a list of all available BanchoBot commands. An example of sending this command is shown below:
 
-*Notice: The `user` parameter for the `REPORT` command is missing.*
+<!--note for editors: the code block below reflects the exact response from banchobot -->
 
 ```
-pippi: !help
-BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
-BanchoBot: WHERE <user>
-BanchoBot: STATS <user>
-BanchoBot: FAQ <item>|list
-BanchoBot: REPORT <reason> - call for an admin
-BanchoBot: REQUEST [list] - shows a random recent mod request
-BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
+13:00 pippi: !help
+13:00 BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+13:00 BanchoBot: WHERE <user>
+13:00 BanchoBot: STATS <user>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <reason> - call for an admin
+13:00 BanchoBot: REQUEST [list] - shows a random recent mod request
+13:00 BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
 ```
 
-### ROLL
+### Roll
 
 ```
-!ROLL <number>
+!roll <argument> <number>
 ```
 
-Draws a random number from 1 to the selected number (default is 100).
-
-#### Usage Example
+`!roll` draws a random number from 1 to the selected number. If a number is not specified or an argument is given, then the maximum number will be set to 100. Examples of sending this command are shown below:
 
 ```
-pippi: !roll 1000
-BanchoBot: pippi rolls 109 point(s)
+13:00 pippi: !roll 1000
+13:00 BanchoBot: pippi rolls 109 point(s)
 ```
 
-### STATS
-
 ```
-!STATS <user>
+13:01 pippi: !roll probability of failure 10
```

Comment:
```suggestion
13:01 pippi: !roll probability of failure
```

Reply:
I added the "10" to show that it doesn't matter what you specific as a max number, bc adding an argument will always result in a number from 1–100

Reply:
I added the "10" to show that it doesn't matter what you specific as a max number, bc adding an argument will always result in a number from 1–100

Reply:
Actually, nvm, idk why i fought this